ten_years_rails.gemspec

```diff
@@ -20,8 +20,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "actionview"
-  spec.add_dependency "activesupport"
+  spec.add_dependency "time_ago_in_words"
```

Comment:
This gem appears to no longer be maintained. What if we approached this as a progressive enhancement? If `actionview` is available, use it. If not, just print the timestamp.

It might look something like this:

```ruby
begin
  require "action_view"
rescue LoadError
  # actionview gem not installed, that's okay.
end

module DateHelpers
  if defined?(ActionView::Helpers::DateHelper)
    include ActionView::Helpers::DateHelper

    def time_ago_in_words(time)
      "#{super(time)} ago"
    end
  else
    # When ActionView isn't available, print ISO8601 formatted string
    def time_ago_in_words(time)
      time.strftime("%Y-%m-%d")
    end
  end
end

class Foo
  include DateHelpers

  def test(time)
    time_ago_in_words(time)
  end
end

puts Foo.new.test(Time.new(2010, 1, 1))
```

This would allow us to remove ActionView from the Gemfile while maintaining the pretty printing for those who have ActionView.

Reply:
I like the progressive approach. I'll tweak my PR to take that into account.